doc/topics/core-structure.md

```diff
@@ -0,0 +1,202 @@
+# Node Core's Internal Structure
+
+This document covers the basics of how Node.js is structured, in an attempt to
+make it easier to contribute to.
+
+Note: The Node.js codebase is in a constant amount of flux, things can and will
+change over time, and so also parts of this document will go out of date over
```

Comment:
Suggestion: get rid of the comma and everything after it and replace it with a separate sentence:

> ...change over time. If you find something in the document that seems incorrect, file [an issue](link to issue tracker here) or a pull request(link to pull request creation form here).